lib/brexit_checker/actions.yaml

```diff
@@ -724,7 +724,11 @@ actions:
     deal Brexit
   guidance_url: https://www.gov.uk/guidance/fresh-fruit-and-vegetable-marketing-standards-if-the-uk-leaves-the-eu-without-a-deal
   criteria:
-  - forestry
+  - any_of:
```

Comment:
It looks like this criteria change has crept in as part of this PR. We should avoid making changes that are unrelated to the card (i.e. T064), as they may not be completed yet, and makes future reasoning about the work we did hard. Can we revert this part of the change for now?

Reply:
Yep, will do!

Reply:
ah ha, so the CSV can cover multiple changes and we need to `git add -p` just out the ones related to a ticket?